silk/config.py

```diff
@@ -28,7 +28,8 @@ class SilkyConfig(metaclass=Singleton):
         'SILKY_PYTHON_PROFILER': False,
         'SILKY_PYTHON_PROFILER_FUNC': None,
         'SILKY_STORAGE_CLASS': 'silk.storage.ProfilerResultStorage',
-        'SILKY_MIDDLEWARE_CLASS': 'silk.middleware.SilkyMiddleware'
+        'SILKY_MIDDLEWARE_CLASS': 'silk.middleware.SilkyMiddleware',
+        'SILKY_JSON_ENSURE_ASCII': True,
```

Comment:
Chinese don't work as Unicode?

Reply:
> Chinese don't work as Unicode?

The Chinese characters are shown below on the page: 
```
"\u8be5\u8f66\u5728\u3010\u53e4\u853a\u5b8f\u6c38\u3011\u68c0\u6d4b\u672a\u5b8c\u6210\uff0c\u65e0\u6cd5\u767b\u5f55"
```

Reply:
did you encoded your chinease as utf-8?

Reply:
> did you encoded your chinease as utf-8?

Yes, the http request send with utf-8 encode